test/contracts-network/colony-network.js

```diff
@@ -287,9 +288,14 @@ contract("Colony Network", accounts => {
     let ensRegistry;
 
     beforeEach(async () => {
+      const USER_HASH = await soliditySha3("user");
+      const COLONY_HASH = await soliditySha3("colony");
```

Comment:
These constants can probably be bumped one scope higher, into the `describe` block.

Reply:
`describe` block is not `async` and these are. Making the `describe` async is problematic I think because of some default mocha behaviour.

Reply:
TIL.

Reply:
`soliditySha3` surely isn't async?

Reply:
Yes good point, the other usages don't `await`.

https://github.com/JoinColony/colonyNetwork/blob/fa71fe33cf34e65ecd0f67cb622c504aa3736f9e/helpers/constants.js#L44

Reply:
Yes you're right, I must be assuming everything in `web3` is `async` since v1. 
I've gone further here and extracted the common logic of setting up a new `ENSRegistrar` into a helper which is reused.
I've also removed the other one reference where we await the `soliditySha3` call.